Dockerfile

```diff
@@ -55,3 +55,79 @@ COPY --from=server_builder /go/pkg/mod/github.com/go-ego /go/pkg/mod/github.com/
 RUN apk add --no-cache --upgrade bc ca-certificates openssl
 RUN mkdir ./modules
 CMD [ "--host", "0.0.0.0", "--port", "8080", "--scheme", "http"]
+
+################################################################################
+# Weaviate CUDA builder
+# Build stage
```

Comment:
Can you please separate the Dockerfile so that the normal `Dockerfile` is left unchanged and a new `Dockerfile.cuda` is create just for an image supporting GPU acceleration.